src/App.vue

```diff
@@ -58,11 +58,15 @@
         <a class="btn btn-primary stretched-link" :href="value._source.url"></a>
       </div>
     </div>
+    <div>
+      <button v-on:click="testInput">Test Input</button>
```

Comment:
Curious why we need this button explicitly. Correct me if I am wrong. Are you suppose to run test cases during next presentation?

Reply:
no, this was just an easy way to run the method inside the vue file